scripts/scenarios/rally/rally-failover-test.yml

```diff
@@ -0,0 +1,55 @@
+{% set flavor_name = flavor_name or "m1.tiny" %}
+{% set image_name = image_name or "cirros-0.3.5-x86_64-tempest-machine" %}
+{% set network_id = network_id or "##net_id##" %}
+NovaServers.boot_and_bounce_server:
+  - 
+    args:
```

Comment:
There are trailing white spaces in this file. Strange that they don't show up here on GitHub, but please remove them.

Reply:
fixed